### ISSUE TITLE
Roller Shutter

### ISSUE THREAD

**Issue by gajdur** [rogro82/io.homeassistant#5]
Hi rogro, any chance to become a Roller support into Homey? Thanks.

**Comment by tccflash** [comment]
That would be pretty nice. If it is possible, please bring the Roller Shutter support in your App.

**Comment by prj32** [comment]
Great idea. I have several blinds (using Automate Pulse 2 Hub) that would benefit from cover type. @tccflash or @gajdur have you developed workaround?

**Comment by mhoogenbosch** [comment]
Yes, would be great if this could be added. I'd love to see my blinds added to Homey.
